### DISCUSSION TITLE
Keep getting error trying to convert using python convert_original_stable_diffusion_to_diffusers.py

### DISCUSSION THREAD

**Discussion by adamwberck** [azuritecoin/OnnxDiffusersUI#21] [Q&A]
Trying to use a custom model but keep getting an error.

`RuntimeError: Attempting to deserialize object on a CUDA device but torch.cuda.is_available() is False. If you are running on a CPU-only machine, please use torch.load with map_location=torch.device('cpu') to map your storages to the CPU.`

Is there a way to set it to use the cpu? CUDA as far as I understand needs a Nvida card,

**Comment by azuritecoin** [comment]
which custom model are you using?
